bibserver/dao.py

```diff
@@ -159,15 +158,15 @@ def bulk_upsert(cls, dataset, state=None):
             else:
                 id_ = make_id(data)
                 data['_id'] = id_
-            
+
             if '_created' not in data:
                 data['_created'] = datetime.now().strftime("%Y%m%d%H%M%S")
             data['_last_modified'] = datetime.now().strftime("%Y%m%d%H%M%S")
-            
+
             index_result = conn.index(data, db, cls.__type__, urllib.quote_plus(id_), bulk=True)
         # refresh required after bulk index
-        conn.refresh()
-    
+        # conn.refresh()
```

Comment:
This looks like it was moved to .indices.refresh().

https://pyes.readthedocs.org/en/latest/changelog.html#deprecated